docs/docsite/rst/iocage_inventory_guide_basics.rst

```diff
@@ -0,0 +1,135 @@
+..
+  Copyright (c) Ansible Project
+  GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+  SPDX-License-Identifier: GPL-3.0-or-later
+
+.. _ansible_collections.community.general.docsite.guide_iocage.iocage_inventory_guide.iocage_inventory_guide_basics:
+
+Basics
+------
+
+As root at the iocage host, create three VNET jails with a DHCP interface from the template
+*ansible_client*:
+
+.. code-block:: console
+
+   shell> iocage create --template ansible_client --name srv_1 bpf=1 dhcp=1 vnet=1
+   srv_1 successfully created!
+   shell> iocage create --template ansible_client --name srv_2 bpf=1 dhcp=1 vnet=1
+   srv_2 successfully created!
+   shell> iocage create --template ansible_client --name srv_3 bpf=1 dhcp=1 vnet=1
+   srv_3 successfully created!
+
+See: `Configuring a VNET Jail <https://iocage.readthedocs.io/en/latest/networking.html#configuring-a-vnet-jail>`_.
+
+As admin at the controller, list the jails:
+
+.. code-block:: console
+
+   shell> ssh admin@10.1.0.73 iocage list -l
+   +------+-------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | JID  | NAME  | BOOT | STATE | TYPE |     RELEASE     |        IP4         | IP6 |    TEMPLATE    | BASEJAIL |
+   +======+=======+======+=======+======+=================+====================+=====+================+==========+
+   | None | srv_1 | off  | down  | jail | 14.2-RELEASE-p3 | DHCP (not running) | -   | ansible_client | no       |
+   +------+-------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | None | srv_2 | off  | down  | jail | 14.2-RELEASE-p3 | DHCP (not running) | -   | ansible_client | no       |
+   +------+-------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+   | None | srv_3 | off  | down  | jail | 14.2-RELEASE-p3 | DHCP (not running) | -   | ansible_client | no       |
+   +------+-------+------+-------+------+-----------------+--------------------+-----+----------------+----------+
+
+Create the inventory configuration:
+
+.. code-block:: console
+
+   shell> cat hosts/02_iocage.yml
```

Comment:
If you use a `console` block to only do `cat /path/to/file` before doing another code block with the file's content, please move the filename into the text:
```suggestion
Create the inventory configuration ``hosts/02_iocage.yml``:

.. code-block:: console

   shell> cat hosts/02_iocage.yml
```
More cases below.